slidey_js.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function() {
+
+
+var pics = [
+"url(http://i.imgur.com/xvQoF0y.jpg)",
+"url(http://i.imgur.com/mlJnuBz.jpg)",
+"url(http://i.imgur.com/hVfr8sa.jpg)",
+"url(http://i.imgur.com/n5C1KCY.jpg)",
+"url(http://i.imgur.com/ZcpGF.jpg)"
+];
+
+var background = $("#background");
+var index = 0;
+function changeImage() {
+  background.css({"background-image": pics[index]});
+  index++
+  if (index === 5) {
```

Comment:
Try using `pics.length` instead of hardcoding a value. That way, you can add more images in the future without having to update your business logic.